Libs/Mesh/Mesh.h

```diff
@@ -3,11 +3,35 @@
 #include "Shapeworks.h"
 #include "ImageUtils.h"
 
+#include <math.h>
+#include <algorithm>
 #include <vector>
 #include <vtkSmartPointer.h>
 #include <vtkPolyData.h>
 #include <string>
 #include <vtkPointData.h>
+#include <vtkCellData.h>
+#include <vtkDataSetAttributes.h>
+#include <vtkSelectPolyData.h>
+#include <vtkClipPolyData.h>
+#include <vtkKdTreePointLocator.h>
+#include <vtkCellLocator.h>
+
+#include <igl/grad.h>
+#include <igl/per_vertex_normals.h>
+#include <geometrycentral/surface/surface_mesh_factories.h>
+#include <geometrycentral/surface/surface_mesh.h>
+#include "geometrycentral/surface/heat_method_distance.h"
+#include "geometrycentral/surface/meshio.h"
+
+#include <vtkLookupTable.h>
+#include <vtkArrowSource.h>
+#include <vtkNamedColors.h>
+#include <vtkPolyDataMapper.h>
+#include <vtkRenderWindow.h>
+#include <vtkRenderWindowInteractor.h>
```

Comment:
Why does Mesh.h need vtkRenderWindow and vtkRenderWindowInteractor?  Mesh.h is included in a lot of places, so extra unneeded headers slow down compilation of the entire codebase.